mrp_subcontracting_inhibit/models/purchase_order.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    subcontracting_inhibit = fields.Boolean(string="Subcontracting inhibit")
```

Comment:
Or call the field the same

```suggestion
    subcontracting_inhibit = fields.Boolean(string="Inhibit subcontracting")
```